include/sound/sof/dai-intel.h

```diff
@@ -46,12 +46,6 @@
 /* bclk idle */
 #define SOF_DAI_INTEL_SSP_CLKCTRL_BCLK_IDLE_HIGH	BIT(5)
 
-/*
- * Max number of DMIC PDM controllers supported
- * Needs to be re-visited for platforms with > 2 controllers.
- */
-#define SOF_DAI_INTEL_DMIC_MAX_PDM	2
-
```

Comment:
need your signoff tag even for reverts. git commit --amend --signoff

Reply:
@plbossart done!